scania-digital-design-system/tegel
# fix: disabled text colors for text field and textarea

## **Describe pull-request**  
Updated text colors in disabled state for Text Field and Textarea. It has been verified with Charlotte that all labels, helper texts and text counters should be grey-400 in light mode and grey-800 in dark mode. Note that not all the text colors have been updated in Figma yet.

## **Issue Linking:**  
- **Jira:** [CDEP-3621](https://tegel.atlassian.net/browse/CDEP-3621)

## **How to test** 
1. Go to preview link below.
2. Check in Text Field and Textarea.
3. Place both components in disabled state and verify that labels, helper text and text counters are grey-400 in light mode and grey-800 in dark mode.

## **Checklist before submission**

## **Suggested test steps**

## **Additional context**  
This PR is a continuation on this closed PR: https://github.com/scania-digital-design-system/tegel/pull/860


[CDEP-3621]: https://tegel.atlassian.net/browse/CDEP-3621?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
